samples/features/Features.NetFive/Features.NetFive.csproj

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
Should we rename this project to Net instead of NetFive?

Reply:
That is a good Idea I will work on that

Reply:
updated